File: energy_frontend/src/app/DTO/LoginResponseDTO.ts
export class LoginResponseDTO {
    username : string;
    token : string;
    userId : number;
    role : string;

    constructor(username : string, token : string, userId : number, role : string) {
        this.username = username;
        this.token = token;
        this.userId = userId;
        this.role = role;
    }
}
  

File: microservices/energy/src/main/java/com/cezar/energy/DTO/LoginResponseDTO.java
package com.cezar.energy.DTO;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class LoginResponseDTO {
    private String username;
    private String token;
    private long userId;
    private String role;
}
